templates/logo-swap-close.html

```diff
@@ -0,0 +1,163 @@
+<style>
+  div.switch-{{ snippet_id }}#brandLogo {
+    background-size: {{ width|default(192, true) }}px {{ height ~ 'px' if height else 'auto' }};
+  }
+
+  .switch-{{ snippet_id }}#brandLogo {
+    height: {{ height|default(192, true) }}px;
+    width: {{ width|default(192, true) }}px;
+  }
+
+  .snippet .icon {
+    top: 10px;
+  }
+
+ {% if clickable or blockable %}
+   .block-snippet-overlay-{{ snippet_id }} {
+       position: absolute;
+       height: 100%;
+       width: 100%;
+       left: 0px;
+       top: 0px;
+       padding-top: 5px;
+       padding-bottom: 5px;
+       z-index: -10;
+       border-radius: 9pt;
+       visibility: hidden;
+       border: solid 0pt white;
+       box-shadow: 0pt 0pt 0pt white;
+   }
+ {% endif %}
+
+ {% if clickable %}
+   .block-snippet-overlay-glow-{{ snippet_id }} {
+       visibility: visible;
+       border: solid 2pt white;
+       box-shadow: 0pt 0pt 10pt #7ac1f9;
+   }
```

Comment:
You can change the above to:
```
.block-snippet-overlay-glow-{{snippet-id}} {
   visibility: visible;
   border: 2px solid fff;
   box-shadow: 0 0 6px 2px #4CB1FF;
}
```
and it will match tiles exactly.